### DISCUSSION TITLE
怎么自动更新

### DISCUSSION THREAD

**Discussion by lhsmm0910** [alex8088/electron-vite#443] [Ideas]
### Clear and concise description of the problem

需要把打包后的xxx.yml和xxx.zip放同一个public目录下还是分开放，http:xxx:3000/public

### Suggested solution

希望出个案例

### Validations

- [X] Follow the [Code of Conduct](https://github.com/alex8088/electron-vite/blob/master/CODE_OF_CONDUCT.md).
- [X] Read the [Contributing Guidelines](https://github.com/alex8088/electron-vite/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://electron-vite.org).
- [X] Check that there isn't [already an issue](https://github.com/alex8088/electron-vite/issues) that requests the same feature to avoid creating a duplicate.

**Comment by HAIlows** [comment]
需要放置在同一文件夹下（http:xxx:3000/public），你需要看yml文件的内容，它引用了什么文件就放什么文件
